ner_v2/detectors/numeral/number_range/README.md

```diff
@@ -0,0 +1,246 @@
+  
+  # Curl output
+  $ {
+      "data": [
+          {
+              "detection": "message",
+              "original_text": "more than 12",
+              "entity_value": {
+                  "max_value": null,
+                  "min_value": "12",
+                  "unit": null
+              },
+              "language": "en"
+          }
+      ]
+  }
+  ```
+
+  
+
+### Steps to add new language for Number Range detection
+
+In order to add any new language you have to follow below steps:
+
+1. Create a directory with `ISO 639-1` code of that language inside `ner_v2/detectors/numeral/number_range/`.  
+
+2. Create a directory named `data` inside the language_code folder.
+
+3. Add a CSV file named `number_range_keywords.csv` inside data folder.  
+
+   Below is the folder structure of same after adding all the files for new language `xy`.
+
+   ```python
+   |__ner_v2
+         |___detectors
+             |___numeral
+                 |___number_range
+                     |___xy    # <- New language Added 
+                     |	  |___data
+                     |      |___number_range_keywords.csv
+                     |      
+                     |__number_detection.py 
+   ```
+
+
+
+
+####  GuideLines to create data files
+
+Below is the brief about how to create data file `number_range_keywords.csv` All the description of the file is explained using Hindi as a reference language. 
+
+1. **number_range_keywords.csv**:  This file contains the vocabs for range keywords, their position around number values and their range type.
+
+   |                        range_variants                        | position | range_type |
+   | :----------------------------------------------------------: | :------: | :--------: |
+   | k uper\| se upar\| se jada \| se adhik \| के ऊपर\|  से ज्यादा \|से जादा \| से अधिक |    1     |    min     |
+   |                     kam se kam\| कम से कम                     |    -1    |    min     |
+   | jyada se jyada \| lagbhag \| ज्यादा से ज्यादा \| जादा से जादा \| लगभग |    -1    |    max     |
+   | se niche \| se kam \| se sasta \| se saste k aas paas\|  k aas pas\| k lagbhag \|  से नीचे \| से कम \| से सस्ता \| से सस्ते \|   के आस पास  \| के लगभग |    1     |    max     |
+   |                           se\|-\|से                           |    0     |  min_max   |
+
+   Here, the 1st column will contain the variants of range keywords separated by pipes, which are present before or after or in between number values and defines what type of number value is present, either min or max. 
+
+   2rd column corresponds to the position of the keyword. If 1 means it will be present after number value, -1 if present before number.  
+
+   3rd column defines the range type of number i.e if it contains min value or max value or both.    
```

Comment:
`3rd column defines the range type of number i.e if it contains min value or max value or both. ` ->

```
3rd column defines what part of the range the variants in the first column appear in the context of. 
- `min`: The variants appear only when mentioning a minimum value of the range
- `max`: The variants appear only when mentioning a maximum value of the range
- `min_max`: The variants appear when mentioning the entire range (both parts)
```